x-pack/plugins/event_log/scripts/mappings.js

```diff
@@ -189,6 +189,18 @@ exports.EcsCustomPropertyMappings = {
       },
     },
   },
+  action: {
```

Comment:
We shouldn't put any new properties at the top-level, as that in theory could collide with some future *standard* ECS property.  For now, we put everything under `kibana`, and I think that would be a fine place for this.

Reply:
Resolved in this commit f933a09f2240f33b542881dcb715e6c7b4a645dc